src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -1296,16 +1318,18 @@ int publish_abort(reservation_t& res) {
       // nothing to abort or already committed/aborted
       continue;
     }
-    const auto& queue_name = topic.cfg.dest.persistent_queue;
+    uint64_t target_shard = topic.shard_id;   
```

Comment:
is shard id `uint64_t` or `uint32_t` ?

Reply:
uint64_t -> as the config param is uint which defaults to uint64_t by default.

Reply:
but it is uint32_t here: https://github.com/ceph/ceph/pull/64018/files#diff-07757d45c279a6208727b04978c761a8a5dc42e27fd2caf111ed1e30d13fa7b7R270
and in other places. 

please make it consistent.